fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -653,7 +687,37 @@ protected void storeSearchParameters(Resource fhirResource, com.ibm.fhir.persist
      * @throws JAXBException
```

Comment:
strange are we actually throwing this?

Reply:
I see this in a bunch of javadoc on FHIRPersistenceJDBCImpl, but i'm guess its just carry-over from "back in the day".  Let me remove it across the board quick...